lighthouse-core/gather/gatherers/accessibility.js

```diff
@@ -50,46 +50,51 @@ function runA11yChecks() {
       'svg-img-alt': {enabled: false},
       'audio-caption': {enabled: false},
     },
-    // @ts-expect-error
-  }).then(axeResults => {
-    // axe just scrolled the page, scroll back to the top of the page so that element positions
-    // are relative to the top of the page
-    document.documentElement.scrollTop = 0;
+  });
+
+  // axe just scrolled the page, scroll back to the top of the page so that element positions
+  // are relative to the top of the page
+  document.documentElement.scrollTop = 0;
+
+  /** @param {import('axe-core/axe').Result} result */
+  const augmentAxeNodes = result => {
+    result.nodes.forEach(node => {
+      // @ts-expect-error - getNodeDetails put into scope via stringification
+      Object.assign(node, getNodeDetails(node.element));
+      // @ts-expect-error - avoid circular JSON concerns
+      node.element = node.any = node.all = node.none = undefined;
+    });
 
+    // Ensure errors can be serialized over the protocol
     // @ts-expect-error
-    const augmentAxeNodes = result => {
+    if (result.error instanceof Error) {
```

Comment:
I added types here. I suppose `.error` is not part of the handwritten, public api.

Reply:
Should verify, I don't think there are any tests for this.